packages/actor-http-retry/lib/ActorHttpRetry.ts

```diff
@@ -0,0 +1,178 @@
+import type { IActionHttp, IActorHttpOutput, IActorHttpArgs, MediatorHttp } from '@comunica/bus-http';
+import { ActorHttp } from '@comunica/bus-http';
+import { KeysHttp } from '@comunica/context-entries';
+import { ActionContextKey, passTest, failTest } from '@comunica/core';
+import type { TestResult } from '@comunica/core';
+import type { IMediatorTypeTime } from '@comunica/mediatortype-time';
+
+export class ActorHttpRetry extends ActorHttp {
+  private readonly mediatorHttp: MediatorHttp;
+  private readonly activeDelays: Record<string, { date: Date; timeout: NodeJS.Timeout }>;
+
+  // Expression that matches dates expressed in the HTTP Date header format
+  // eslint-disable-next-line max-len
+  private static readonly dateRegex = /^(Mon|Tue|Wed|Thu|Fri|Sat|Sun), [0-9]{2} (Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec) [0-9]{4} [0-9]{2}:[0-9]{2}:[0-9]{2} GMT$/u;
+  private static readonly numberRegex = /^[0-9]+$/u;
+  private static readonly keyWrapped = new ActionContextKey<boolean>('urn:comunica:actor-http-retry#wrapped');
+
+  public constructor(args: IActorHttpQueueArgs) {
+    super(args);
+    this.mediatorHttp = args.mediatorHttp;
+    this.activeDelays = {};
+  }
+
+  public async test(action: IActionHttp): Promise<TestResult<IMediatorTypeTime>> {
+    if (action.context.has(ActorHttpRetry.keyWrapped)) {
+      return failTest(`${this.name} can only wrap a request once`);
+    }
+    const retryCount = action.context.get<number>(KeysHttp.httpRetryCount);
```

Comment:
This should now work:

```suggestion
    const retryCount = action.context.get(KeysHttp.httpRetryCount);
```